Tests/Packet++Test/main.cpp

```diff
@@ -7716,6 +7735,7 @@ int main(int argc, char* argv[]) {
 	PTF_RUN_TEST(TcpPacketWithOptionsParsing2, "tcp");
 	PTF_RUN_TEST(TcpPacketCreation, "tcp");
 	PTF_RUN_TEST(TcpPacketCreation2, "tcp");
+	PTF_RUN_TEST(TcpPacketParsing, "tcp");

```

Comment:
I'd suggest changing the test-case name to `TcpMalformedPacketParsing` or something similar because there are already test-cases that test TCP packet parsing: `TcpPacketNoOptionsParsing`, `TcpPacketWithOptionsParsing`, `TcpPacketWithOptionsParsing2`

Reply:
Done